pkg/api/controller.go

```diff
@@ -3401,7 +3443,10 @@ func NewController(
 	auditChecker AuditChecker,
 	logger logging.Logger,
 	emailer *email.Emailer,
+	oidcAuthenticator *oidc.Authenticator,
+	sessionStore sessions.Store,
 ) *Controller {
+	gob.Register(oidc.Claims{})
```

Comment:
Synced f2f: not really an option here since internally gorilla sessions have a `map[interface{}]interface{}` which is not support by the JSON encoder.